CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## Unreleased
```

Comment:
Can you please call this `0.19.4` (and add the link to its not-yet-existing release notes at the bottom)? Then we can cut a release whenever we like, without an additional push to `main` to give this the correct name. If we're wrong about the name of the next release (and we decide its actually `0.20.0`) then we'll still need a renaming commit, but its no worse than if we'd had `Unreleased`.